cedar-policy/CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ Starting with version 4.0, changes marked with a star (*) are _language breaking
 Cedar Language Version: 4.0
 
 ### Added
+- A new experimental API (`compute_entity_manifest`)
+  that provides the Entity Manifest: a data
+  structure that describes what data is required to satisfy a 
+  Cedar request. Entity Manifests improve Cedar performance dramatically
+  in a safe way.
```

Comment:
Probably not putting the performance claim in the changelog. Among other things it's not clear what we're comparing against for the "improve performance dramatically".  Vs sending all entities?  Are users actually doing that?  I suspect they aren't, and are instead rolling their own slicing, in which case this is more about improving safety than performance

Reply:
I'd also link to the RFC in the changelog entry

Reply:
Good idea